### ISSUE TITLE
If used, process can't be exited

### ISSUE THREAD

**Issue by Diluka** [Federerer/Notifications.Wpf#10]
window process is still running background. how to kill them?

### UPDATE
Only happens when showing outside window

**Comment by MuhammadAliYousaf** [comment]
I am also getting same issue, any solution for that?

**Comment by Diluka** [comment]
@alisolutions Yes. exit on last window close will not work. Shutdown app manually by calling `App.Shutdown()`

**Comment by chrisstayte** [comment]
Did this get a solution?

**Comment by Xinnony** [comment]
Or `Environment.Exit(0);`
Close this Issue ?

**Comment by srehmanali** [comment]
Is someone working on it?

**Comment by tobster-de** [comment]
There's already a fix. Someone just needs to merge #13.

**Comment by xmoer** [comment]
Based on this fix: https://github.com/Federerer/Notifications.Wpf/pull/13/files

Document (Controls/Notification.cs) line 116~117:
```
if (notificationCount == 0)
                currentWindow?.Hide();
```
Revised to:
```
if (notificationCount == 0)
                currentWindow?.Close();
```
In this way, you can exit normally.

**Comment by xmoer** [comment]
Based on this fix: https://github.com/Federerer/Notifications.Wpf/pull/13/files
Modify the following files:

Document "Notifications.Wpf/Controls/Notification.cs" line 116~117:
```
if (notificationCount == 0)
    currentWindow?.Close();
```
Document "Notifications.Wpf/NotificationManager.cs" line 53~54
```
if (!_window.IsInitialized || !_window.IsLoaded)
{
    var workArea = SystemParameters.WorkArea;

    _window = new NotificationsOverlayWindow
    {
        Left = workArea.Left,
        Top = workArea.Top,
        Width = workArea.Width,
        Height = workArea.Height
    };
}

if (Areas != null && !_window.IsVisible)
{
    _window.Show();
}
```

**Comment by tobster-de** [comment]
The fork [Platonenkov/Notifications.Wpf](https://github.com/Platonenkov/Notifications.Wpf) is more active and already included the mentionend fix. It also provides an up-to-date Nuget.

**Comment by GeorgeFotopoulos** [comment]
> The fork [Platonenkov/Notifications.Wpf](https://github.com/Platonenkov/Notifications.Wpf) is more active and already included the mentionend fix. It also provides an up-to-date Nuget.

This fork doesn't include the fix mentioned by user xmoer above you. It's sad because it's quite easy fix and would help a ton if he would take the time to include the code in his project.

**Comment by Platonenkov** [comment]
> > The fork [Platonenkov/Notifications.Wpf](https://github.com/Platonenkov/Notifications.Wpf) is more active and already included the mentionend fix. It also provides an up-to-date Nuget.
> 
> This fork doesn't include the fix mentioned by user xmoer above you. It's sad because it's quite easy fix and would help a ton if he would take the time to include the code in his project.

it is an open project, anyone can make changes if they find an error, if it really need.

**Comment by Platonenkov** [comment]
so,  theare was second problem, that after window was closed it never been null becouse it is static.. it was resolved in second commit.
in this case, there was an error trying to open an already closed window
